reference/halyard/high-availability.md

```diff
@@ -135,6 +135,14 @@ This service's name when [configuring its sizing](/reference/halyard/component-s
 
 To add a [custom profile](/reference/halyard/custom/#custom-profiles) or [custom service settings](/reference/halyard/custom/#custom-service-settings) for this service, use the name `echo-worker`.
 
+## Deleting Orphan Services
+
+When enabling or disabling HA for a service on a running Spinnaker, Halyard will not clean up the old service(s) by default. This means that if a non-HA Clouddriver is running and Spinnaker is then deployed with HA Clouddriver enabled, the non-HA Clouddriver will still be running, even though it is no longer used. To clean up these orphan services, add a `--delete-orphan-services` flag to `hal deploy apply`:
```

Comment:
The first sentence seems to  contradict the rest of the description. Mostly that's probably my lack of understanding, but if Halyard is going to clean up old services, I would not expect the non-HA Clouddriver to be still running. Could you clarify?

Reply:
nm, I read "Halyard will clean up" rather than "Halyard will not clean up."

Carry on.